nuget/nunitlite.nuspec

```diff
@@ -37,10 +37,15 @@ How to use this package:
     <file src="NOTICES.txt" />
     <file src="CHANGES.txt" />
     <file src="bin/net-2.0/nunitlite.dll" target="lib/net20" />
+    <file src="bin/net-2.0/nunitlite-runner.exe" target="lib/net20" />
     <file src="bin/net-4.0/nunitlite.dll" target="lib/net40" />
+    <file src="bin/net-4.0/nunitlite-runner.exe" target="lib/net40" />
     <file src="bin/net-4.5/nunitlite.dll" target="lib/net45" />
+    <file src="bin/net-4.5/nunitlite-runner.exe" target="lib/net45" />
     <file src="bin/portable/nunitlite.dll" target="lib/dotnet" />
+    <file src="bin/portable/nunitlite-runner.exe" target="lib/dotnet" />
     <file src="bin/portable/nunitlite.dll" target="lib/portable-net45+win8+wp8+wpa81+Xamarin.Mac" />
+    <file src="bin/portable/nunitlite-runner.exe" target="lib/portable-net45+win8+wp8+wpa81+Xamarin.Mac" />
```

Comment:
The runner will only run portable tests if the tests themselves are portable. That may be the case in some situations, but so far, almost all of the testing I am doing involves tests compiles for one of the platforms that portable supports. In that case, I am pretty sure the .NET 4.5 based runner will not run them.

If they do create portable tests, then the runner in the .net 4.5 directory will run them.

Thinking about it that way, I think all the runners should be in a tools directory with platform specific sub-directories.

Your comment on Xamarin is actually quite prescient. I wrote a blog post on [Testing Xamarin Projects using NUnit 3](http://www.alteridem.net/2015/12/21/testing-xamarin-projects-using-nunit-3/) today. In it, I am advocating that users create a shared project for there tests and reference it from a .NET 4.5 test project and from platform specific test projects. The .NET 4.5 project can then run the majority of tests in a CI environment, and the platform specific tests can add the ability to run the tests on the platform and additionally run a few platform specific tests. Currently working in Xamarin, I am seeing 90 to 95% code reuse across platforms and my tests also reflect that ratio.